source/_integrations/binary_sensor.template.markdown

```diff
@@ -294,3 +293,45 @@ binary_sensor:
 ```
 
 {% endraw %}
+
+### Working without entities
+
+The `template` sensors are not limited to use attributes from other entities but can also work with [Home Assistant's template extensions](/docs/configuration/templating/#home-assistant-template-extensions). If the template includes some non-deterministic property such as time in its calculation, the result will not continually update, but will only update when some entity referenced by the template updates. 
+
+There's a couple of options to manage this issue. This first example creates a `sensor.time` from the [Time & Date](/integrations/time_date/) component which updates every minute, and the binary sensor is triggered by this updating. The binary sensor returns true if in the first half of the hour:
+
+{% raw %}
+```yaml
+sensor:
+  - platform: time_date
+    display_options:
+      - 'time'
+
+binary_sensor:
+  - platform: template
+    sensors:
+      half_hour:
+        value_template: '{{ (states.sensor.time.state[3:]|int) < 30 }}'
+```
+{% endraw %}
+
+An alternative to this is to create an interval-based automation that calls the service `homeassistant.update_entity` for the entities requiring updates. This modified example updates every 2 minutes:
```

Comment:
This example is slightly confusing because it only needs to update 2 times per hour but it actually updates every 2 minutes so now I wonder whether those two numbers are related.

Reply:
It is also updating `sensor.half_hour` which should be `binary_sensor.half_hour`.

I'll adjust it to be less confusing